### PR TITLE
DOC-1714: Updated the database-set-up.md file for CENM 1.3, 1.4 and 1.5

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by ramkerai — DISMISSED** [review]

File: content/en/docs/cenm/1.3/database-set-up.md [hunk 1]
```diff
@@ -327,37 +327,38 @@ GRANT SELECT ON my_admin_user.DATABASECHANGELOG to my_user;
...
-GRANT SELECT, INSERT, UPDATE, DELETE ON TRUST_STORE;
-GRANT SELECT, INSERT, UPDATE, DELETE ON WHITELIST_CONTRACTS_CONFIG;
-GRANT SELECT, INSERT, UPDATE, DELETE ON WHITELIST_CONTRACTS_CORDAPPS;
-GRANT SELECT, INSERT, UPDATE, DELETE ON WHITELIST_CONTRACTS_EXCLUDE;
-GRANT SELECT, INSERT, UPDATE, DELETE ON WHITELIST_CONTRACT_ATTACH_IDS;
-GRANT SELECT, INSERT, UPDATE, DELETE ON WHITELIST_CONTRACT_CONFIG;
-GRANT SELECT, INSERT, UPDATE, DELETE ON WORKFLOW_CONFIG;
-GRANT SELECT, INSERT, UPDATE, DELETE ON WORKFLOW_CONFIG_CRL_FILES;
-GRANT SELECT, INSERT, UPDATE, DELETE ON ZONE;
+RANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.CONFIGURATION_METADATA to my_user;
```
Comment (Kirill-Anisimov) [inline content/en/docs/cenm/1.3/database-set-up.md:330]: I think "G" escaped here.

File: content/en/docs/cenm/1.5/database-set-up.md [hunk 2]
```diff
@@ -369,23 +369,23 @@ CREATE USER my_user identified by my_password;
...
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.GROUP_DATA;
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.OAUTH_REFRESH_TOKENS;
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.PERMISSIONS;
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.PERMISSION_ASSIGNMENTS;
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.ROLES;
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.ROLE_ASSIGNMENTS;
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.USERS;
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.USERS_IN_GROUPS;
-GRANT SELECT, INSERT, UPDATE, DELETE ON my_admin_user.USER_DATA;
+GRANT SELECT ON my_admin_user.SEQ_AUDIT_EVENTS  to my_user;
```
Comment (Kirill-Anisimov) [inline content/en/docs/cenm/1.5/database-set-up.md:372]: Seems like there is an extra space before "to".

**Review by Kirill-Anisimov — CHANGES_REQUESTED** [review]
Hi @AnnaK-R3 ! I left two inline comments one for 1.3 (cool that you spotted changes were needed there too) version and another one for 1.5. Just a tiny little fixes. Except this, everything looks good to me.
Thank you!

File: content/en/docs/cenm/1.5/database-set-up.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/docs/cenm/1.5/database-set-up.md]: Deleted the extra space.

File: content/en/docs/cenm/1.3/database-set-up.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/docs/cenm/1.3/database-set-up.md]: Added the missing "G".

**Comment by AnnaK-R3** [comment]
> Hi @AnnaK-R3 ! I left two inline comments one for 1.3 (cool that you spotted changes were needed there too) version and another one for 1.5. Just a tiny little fixes. Except this, everything looks good to me.
> Thank you!

@Kirill-Anisimov Thanks for the comments! Applied and pushed updates to the PR.

**Review by Kirill-Anisimov — APPROVED** [review]
Looks good!

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**